### ISSUE TITLE
Version 0.19.1 does not compile on Solaris 10

### ISSUE THREAD

**Issue by vsimko** [tgouhier/biwavelet#29]
Errors are here:
https://www.r-project.org/nosvn/R.check/r-patched-solaris-sparc/biwavelet-00install.html

I'll first install R inside Solaris 10. This will be a x86 VM installed through vagrant and running in my VirtualBox:

``` sh
vagrant init fiftycuatro/solaris10-u11
vagrant up --provider virtualbox
```

Now, I can login to the Solaris VM and install R

``` sh
vagrant ssh
# now inside Solaris VM
sudo pkgutil -i r_base
```

For C/C++ compiler, I need Solaris Studio downloaded from here:
http://www.oracle.com/technetwork/server-storage/solarisstudio/downloads/index-jsp-141149.html

Copy the downloaded archive to VM and install:

``` sh
bzcat SolarisStudio12.4-solaris-x86-pkg.tar.bz2 | /bin/tar -xf -
```

**Comment by vsimko** [comment]
useful stackoverflow page: http://stackoverflow.com/questions/24094963

**Comment by vsimko** [comment]
it seems that this issue has been fixed together with issue #33

Results for **r-patched-solaris-x86**:
https://www.r-project.org/nosvn/R.check/r-patched-solaris-x86/biwavelet-00check.html

... still waiting for **r-patched-solaris-sparc** results.

**Comment by vsimko** [comment]
Works on spark as well
https://www.r-project.org/nosvn/R.check/r-patched-solaris-sparc/biwavelet-00check.html
